core/src/main/java/novoda/lib/sqliteprovider/provider/action/InsertHelper.java

```diff
@@ -22,16 +24,18 @@
  */
 public class InsertHelper {
 
-    private final ExtendedSQLiteOpenHelper dbHelper;
+    private final SQLiteOpenHelper dbHelper;
+    private final SQLiteDatabaseMetaInfo metaInfo;
 
-    public InsertHelper(ExtendedSQLiteOpenHelper helper) {
-        this.dbHelper = helper;
+    public InsertHelper(SQLiteOpenHelper databaseHelper, SQLiteDatabaseMetaInfo metaInfo) {
```

Comment:
These changes require a major release ⚠️  -- it's fine since #69 and #70 will require one anyway